src/Operation/Aggregate.php

```diff
@@ -246,8 +246,8 @@ public function __construct($databaseName, $collectionName, array $pipeline, arr
             unset($options['batchSize']);
         }
 
-        $this->databaseName = (string) $databaseName;
-        $this->collectionName = isset($collectionName) ? (string) $collectionName : null;
+        $this->databaseName = $databaseName;
+        $this->collectionName = $collectionName ?? null;
```

Comment:
If the `$collectionName` param is now a nullable string, I think we can omit `??` here for the same effect. The ternary was only necessary to restrict the `(string)` cast for non-null values.

Reply:
Good point. Removed the unnecessary fallback to `null`.